data_pipeline_api/registry/access_upload.py

```diff
@@ -298,12 +243,82 @@ def unique_posts(posts: List[YamlDict]) -> List[YamlDict]:
     return [yaml.safe_load(t) for t in set_of_yamls.keys()]
 
 
+def _get_accessibility(config):
+    if all(int(event.get("accessibility", 0)) == 0 for event in config["io"]):
+        return 0
+    else:
+        return max(int(event.get("accessibility", 0)) for event in config["io"])
+
+
+def _add_code_repo(posts: List[YamlDict], model_name: str, model_git_sha: str, model_uri: str) -> YamlDict:
+    """
+    Creates a code_repo_release and adds it to the list of objects to post to the data registry
+
+    :param posts: List of posts to the data registry, will be modified
+    :param model_name: name of the model
+    :param model_git_sha: git sha of the model
+    :param model_uri: uri that the model is located at
+    :return: the object reference to the code repo release
+    """
+    code_repo_obj = _create_target_data_dict(
+        DataRegistryTarget.object, {DataRegistryField.description: f"{model_name}+{model_git_sha}"}
+    )
+    code_repo_release = _create_target_data_dict(
+        DataRegistryTarget.code_repo_release,
+        {
+            DataRegistryField.name: model_name,
+            DataRegistryField.version: model_git_sha,
```

Comment:
The database version field seems to be a semver, so I'm not sure if this is correct. [This conversation](https://zulip.scrc.uk/#narrow/stream/14-Data/topic/Python.20API/near/16815) is related.

Reply:
is this something I'll be able to read out the access yaml?

Reply:
I guess it's the model_version that I was reading before